src/front-end/.gitignore

```diff
@@ -0,0 +1,149 @@
+# Created by https://www.toptal.com/developers/gitignore/api/angular,intellij+all,windows
```

Comment:
Remove this file, we won't be using it I think.
All the front end related code should be inside front-end/*